bindings/pyroot_experimental/PyROOT/src/CppCallablePyz.cxx

```diff
@@ -692,7 +692,7 @@ static PyMethodDef CallableImplMethods[] =
 {
     {"__init__", (PyCFunction)GenericCallableImpl_init, METH_VARARGS|METH_KEYWORDS, "Parse decorator arguments"},
     {"__call__", ProxyCallableImpl_call, METH_VARARGS, "Create C++ wrapper function"},
-    {NULL},
+    {NULL, NULL, 0, NULL}
```

Comment:
Just to be more specific (this can help us in the future): `missing-field-initializers` is the exact name of the warning and we saw it with gcc 7 & 8. I would add it to the commit message